CarbStor_Community_Models/Community_Model_Information.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+#add in pathlib to make sure everyone can get this data
+
+import pandas as pd
+from cobra.io import read_sbml_model
+from pathlib import Path
+
+from micom import Community
+from micom.workflows import build, grow, tradeoff, fix_medium,build_database
+from micom import load_pickle
+from micom.viz import plot_tradeoff, plot_exchanges_per_sample, plot_growth
+
+
+#set variables for importing data
+HERE = Path(__file__).parent.resolve()
+ROOT = HERE.parent.resolve()
+COM_MODEL = HERE.joinpath("One.pickle")
```

Comment:
I think this needs an additional folder:

```python
COM_MODEL = HERE.joinpath("CarbStor_Community/One.pickle"
```

Reply:
Also I think I messed you up with giving the ROOT, I wasn't sure where that file was being run from. Try something like this:
```python
# set variables for importing data
HERE = Path(__file__).parent.resolve()
COM_MODEL = HERE.joinpath("Carbstor_Community/One.pickle")
NO_CURTO = HERE.joinpath("CarbStor_Community_NoCurtobacterium/One.pickle")
```